### PR TITLE
feat(content-band-list-block): Section content band list block.

### PR DESCRIPTION
**Pull request by nimithshetty17**
## Issue

Fixes #[MERATIVE-781](https://jira.sdlc.merative.com/browse/MERATIVE-781)

## Description

This ticket is part of the phase 3 effort, which is to migrate all of the remaining components/patterns in AEM over to Franklin. Sections > Section Content band list
Note: part of this work is to clean up the naming in Franklin to ensure it aligns with our Design Systems.

**New**
image

## Design Specs

> If applicable, add the direct link to the design specs of the component/feature that's part of this PR.

- Figma Link - https://www.figma.com/file/vw24IPXXt4vCh9BNumka9A/Web--Merative-Digital-Design-System-2.0?type=design&node-id=1459-14993&mode=design&t=WtS8KBTRL2E1rFnO-0

## Test URLs
  
https://section-content-band-list--merative2--nimithshetty17.hlx.page/block-library/sections/section-content-band-list
  
## Testing Instruction

To test the content band list and its responsiveness with respect to the design across all devices.

### CODE REVIEW

**Comment by sachinmesh** [comment]
Checked, its good to go @nimithshetty17 

CC: @proeung @keith-kaplan @helms-charity @sharathmrft @Shalini-SB

**Review by sachinmesh — APPROVED** [review]

**Review by sharathmrft — APPROVED** [review]
@nimithshetty17 
Back ground color for each content Band List  is rendering properly. Gradient, Bone and white color are appearing as per the design and Responsiveness also verified it is fine.
@proeung @sachinmesh @Shalini-SB @keith-kaplan

File: blocks/content-intro/content-intro.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      order: 1;
+      max-width: 648px;
+      margin-right: 60px;
+  }
+
+  .section-content-band-list.text-on-right .content-intro-wrapper>div p:not(:first-child):not(.button-container) {
+      order: 2;
+      max-width: 357px;
+      margin-left: 60px;
+  }
```
Comment (proeung) [inline blocks/content-intro/content-intro.css:84]: This is incorrect. We don't need the option to make the paragraph to be on the right-hand side. The purpose of this work is to be able to apply text styling to the right side column to match the design (see attached). The headline and short description should stay within the first column and the side text should be displayed when added to the second column.

Screenshot 2023-07-27 at 9 36 02 AM

**PR preview**

Screenshot 2023-07-27 at 9 36 43 AM

**Review by proeung — CHANGES_REQUESTED** [review]
See the comment above. Also, I'm happy to open a PR with the proper changes that we should add to this section.

cc: @keith-kaplan

File: blocks/content-intro/content-intro.css (hunk 1 above)
Comment (keith-kaplan) [reply blocks/content-intro/content-intro.css:84]: @proeung @nimithshetty17  I agree. The paragraph text should be in the right column and there should be now paragraph in the left column. I've updated the authoring doc to make the headline longer for the proper use case here.

https://section-content-band-list--merative2--nimithshetty17.hlx.page/block-library/sections/section-content-band-list

**Comment by nimithshetty17** [comment]
@proeung @keith-kaplan - I have made changes to the styles accordingly. Please have a look and let me know if there are any concerns.

**Review by proeung — CHANGES_REQUESTED** [review]
@nimithshetty17 I'm still seeing issues within this PR.

1. The container of the container that's wrapper the first column should not be 100%. Please see the design.
Screenshot 2023-07-31 at 3 22 37 PM

2. As mentioned above, there's no need to have a class `.text-on-right` on the parent element area. Styling for the second column can be set in CSS or you can conditionally add a classname if the value exists within that column.
Screenshot 2023-07-31 at 3 20 43 PM

**Comment by proeung** [comment]
@nimithshetty17 In an effort to move this ticket along. I'm gonna go ahead and take over this work. I'll assign you as one of the reviewers when it's ready.

**Pull request closed without merging**